guides/common/modules/ref_utilities-for-collecting-log-information.adoc

```diff
@@ -15,27 +20,31 @@ Additionally, the `foreman-debug` command exports tasks run during the last 60 d
 By default, the output tar file is located at `/tmp/__task-export-xxx.tar.xz__`.
 If the file is missing, see the file `/tmp/task-export.log` to learn why task export was unsuccessful.
 
-For more information, run `foreman-debug --help`.
+ifndef::orcharhino[]
+For more information, run `foreman-debug -h`.
+endif::[]
+ifdef::orcharhino[]
+For more information, run `orcharhino-debug -h`.
+endif::[]
 
 There is no timeout when running this command.
+ifdef::satellite[]
 |`sosreport` |The `sosreport` command is a tool that collects configuration and diagnostic information from a Red{nbsp}Hat Enterprise{nbsp}Linux system, such as the running kernel version, loaded modules, and system and service configuration files.
 The command also runs external programs (for example: `foreman-debug -g`) to collect {Project}-specific information, and stores this output in a tar file.
```

Comment:
I started with this since the example is no longer correct. Now I'm wondering if we can't simplify this to:
```suggestion
|`sosreport` |The `sosreport` command is a tool that collects configuration and diagnostic information from a Linux system, such as the running kernel version, loaded modules, running services, and system and service configuration files. This output is stored in a tar file.
```

Reply:
Applied.